### DISCUSSION TITLE
How are nested tags handled (ol inside a div)?

### DISCUSSION THREAD

**Discussion by bharatn** [codex-team/editor.js#1920] [Q&A]
Hey EditorJS team,

First of all. great tool. I have more faith in this after abandoning draftJS and slateJS.

I have this html

```
<div class="classNameX">
		<ol>...
</div>
```

I am using the list plugin

The rendered HTML is

```
<div class="ce-block__content">
     <ol class="cdx-block cdx-list cdx-list--ordered"....
```

ol tag is fine but my div tag with the className is missing in the generated HTML.

div tag with className is also missing in the save output. ol is fine.

```
"<ol><li>....."
```

Am i missing another plugin to handle div's? essentially I have few divs in the html and none of them are in the render or save output.

Thank you

**Comment by RocMark** [comment]
same here
